src/style.css

```diff
@@ -1 +1,94 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir, Arial, sans-serif;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy, Arial, sans-serif;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+.body {
+  margin: 0;
+  padding: 0 8px;
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+}
+
+.search-bar {
+  display: flex;
+  flex-direction: column;
+}
+
+.input {
+  box-sizing: border-box;
+  display: flex;
+  align-items: center;
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+
+  color: #3d4e61;
+}
+
+.search-bar__big {
+  height: 70px;
+  font-size: 16px;
+  line-height: 22px;
+  padding: 25px 0 26px 62px;
+  background: url('./images/Search.svg') no-repeat 26px center;
+
+  background-size: 19px;
+  margin-block: 20px;
+}
+
+.search-bar__small {
+  height: 42px;
+  font-size: 14px;
+  padding-left: 33px;
+  background: url('./images/Search.svg') no-repeat 13px center;
+  background-size: 11px;
+}
+
+.input::placeholder {
+  color: #3d4e61;
+  font-weight: 300;
+  text-shadow: none;
+}
+
+.search-bar__big::placeholder {
+  font-size: 16px;
+}
+
+.search-bar__small::placeholder {
+  font-size: 14px;
+}
+
+.input:hover {
+  box-shadow: 0 4px 4px 0 #00000040;
+  border: 1px solid #e1e7ed;
+}
+
+.search-bar__big:focus {
+  color: #3d4e61;
+  text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.3);
+  font-size: 14px;
+  line-height: 22px;
+
+  outline: none;
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
```

Comment:
The `font-size` and `line-height` properties in this block are the same as in the `.search-bar__big` class. There's no need to repeat them here, they will be inherited from the `.search-bar__big` class.